modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/data/provider/test/DDMRESTDataProviderTest.java

```diff
@@ -615,6 +646,27 @@ private void _setUserPermissionChecker(boolean guest) throws Exception {
 			PermissionCheckerFactoryUtil.create(user));
 	}
 
+	private void _testGetDataWithLocale(
+			long ddmDataProviderId, Locale locale, String outputParameterId)
+		throws Exception {
+
+		DDMDataProviderRequest ddmDataProviderRequest =
+			_createDDMDataProviderRequest(
+				ddmDataProviderId, null, locale, null, null, null, null);
+
+		DDMDataProviderResponse ddmDataProviderResponse =
+			_ddmDataProvider.getData(ddmDataProviderRequest);
+
+		List<KeyValuePair> keyValuePairs = ddmDataProviderResponse.getOutput(
+			outputParameterId, List.class);
+
+		for (KeyValuePair keyValuePair : keyValuePairs) {
```

Comment:
@p-albuquerque please inline

```
for (KeyValuePair keyValuePair :
		(List<KeyValuePair>)ddmDataProviderResponse.getOutput(
			outputParameterId, List.class)) {
```

Reply:
Ops, I checked the tests again, I need to do a little adjust here, because this way have causing a test compile fail:
![image](https://github.com/liferay-forms/liferay-portal/assets/79872134/22e3ecf0-38ea-4d88-85f3-ca4282c33bad)